app/templates/direct-award/end-search.html

```diff
@@ -29,49 +29,61 @@
 {% endblock %}
 
 {% block main_content %}
+{% if errors %}
+  <div class="validation-masthead" aria-labelledby="validation-masthead-heading" aria-role="group" tabindex="-1">
+    <p>
+      You need to <a href="#user_understands">confirm that you’ve finished editing your search</a> before you
+      export your results.
+    </p>
+  </div>
+{% endif %}
```

Comment:
This feels like it should be using the validation masthead template from the frontend toolkit?

Reply:
It's based on that, but this is an intentional change because the form only has one question. So we don't need the 'heading plus list of question/errors' format, but something slightly different i.e. just one para of text.

Reply:
We use the standard heading+question for all the other one-question forms.

Reply:
Well the login screen has something different. The content here seems right to me, and has been discussed a fair amount.